pkg/virt-controller/watch/vmi_test.go

```diff
@@ -1497,7 +1497,7 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 			setReadyCondition(vmi, k8sv1.ConditionTrue, "")
 			vmi.Status.Phase = virtv1.Running
 			pod := NewPodForVirtualMachine(vmi, k8sv1.PodRunning)
-			pod.Status.Conditions = []k8sv1.PodCondition{{Type: k8sv1.PodReady, Status: k8sv1.ConditionTrue}}
+			pod.Status.Conditions = append(pod.Status.Conditions, k8sv1.PodCondition{Type: k8sv1.PodReady, Status: k8sv1.ConditionTrue})
```

Comment:
I think it would be great to also get a negativ that the condition gest added with `ConditionFalse`.

Reply:
Unit test added in bb1913f.